interpreter.py

```diff
@@ -181,24 +202,48 @@ def exec_operation(self, instruction):
             var_val = self.lookup_var(args[0])
             assert isinstance(var_val, Integer)
             return Float(float(var_val.value))
+        elif opcode == LLVMBr:
+            if LLVMIsConditional(instruction):
```

Comment:
Can you add a comment explaining why instruction would not be a conditional? It's not obvious at first glance.